recipes-security/refpolicy/refpolicy-mcs-2.%/patches/policy.modules.services.acpi.diff

```diff
@@ -1,14 +1,13 @@
 # apmd now handled in acpi.te
 --- a/policy/modules/services/acpi.te
 +++ b/policy/modules/services/acpi.te
-@@ -245,3 +245,10 @@ optional_policy(`
+@@ -235,3 +235,9 @@ optional_policy(`
  optional_policy(`
  	xserver_domtrans(acpid_t)
  ')
 +
-+########################################
-+#
-+# OpenXT local policy
-+#
-+# Early devtmpfs, before udev relabel.
-+dev_dontaudit_rw_generic_chr_files(apmd_t)
++gen_require(`
++    type device_t;
++')
++
++allow acpid_t device_t:dir { watch };
```

Comment:
I think this should be:
dev_watch_dev_dirs(acpid_t);

And then you don't need the gen_require.

Reply:
Done.